docs/content/any/project/changelogs/NEXT.md

```diff
@@ -30,6 +30,10 @@ draft: true
   before upgrading.
 {{% /callout %}}
 
+* Previously we supported `POLAR_LOG=trace` and `POLAR_LOG=1` which enabled verbose "TRACE"-level logging of the execution of queries and their constituent goals within the Polar VM.
+* `POLAR_LOG=on` was slightly less verbose than `POLAR_LOG=trace` but still produced a voluminous output which made it hard to parse and follow the execution of a particular query.
+* To enable easier query debugging we have broken out `POLAR_LOG` into new discrete `INFO` and `TRACE` levels. Specifying `POLAR_LOG=info` will cause Polar to emit a more concise log output intended to be consumed by developers as they build and debug their Polar policies. The more verbose TRACE output is still available through `POLAR_LOG=trace`. Check out our [documentation](/reference/tooling/tracing.html) for more information on tracing.
```

Comment:
Random thought: do we care about casing? since we're referring to the levels as `INFO` and `TRACE` might someone expect that `POLAR_LOG=TRACE` would give them trace output instead of info?

Reply:
updated to be case agnostic with respect to specifying log levels